test/DynamoCoreWpfTests/ConverterTests.cs

```diff
@@ -613,27 +613,6 @@ public void FilePathDisplayConverterInternalDoesNotThrow()
             Assert.DoesNotThrow(() => FilePathDisplayConverter.ShortenNestedFilePath(@"\\psf\\Home\somedyn.dyn"));
             Assert.DoesNotThrow(() => FilePathDisplayConverter.ShortenNestedFilePath(@"\\psf\somedyn.dyn"));
             Assert.AreEqual(@"\\psf\Home\Desktop\somedyn.dyn", FilePathDisplayConverter.ShortenNestedFilePath(@"\\psf\Home\Desktop\somedyn.dyn"));
-        }
-
-        [Test]
-        public void TreeViewListBoxTitleConverterTest()
-        {
-            var headerStrip = new HeaderStrip();
-            List<HeaderStripItem> headerItems = new List<HeaderStripItem>();
-            headerItems.Add(new HeaderStripItem() { Text = "CREATE" });
-            headerStrip.HeaderStripItems = headerItems;
-            var listBox = new ListBox();
-            listBox.Items.Add("Test");
-
-            object[] objects = new object[2];
-            objects[0] = headerStrip;
-            objects[1] = listBox;
-            
-            var converter = new TreeViewListBoxTitleConverter();
-            converter.Convert(objects, null, null, null);
-
-            var item = headerItems.FirstOrDefault();
-            Assert.AreEqual("C", item.Text);
-        }
```

Comment:
this test is not required. this test is for the create action query labels. We dont have those labels now.